doc/kconfig/conf.py

```diff
@@ -44,6 +45,29 @@
 ncs_cache_config = NRF_BASE / "doc" / "cache.yml"
 ncs_cache_manifest = NRF_BASE / "west.yml"
 
+# Options for external_content -------------------------------------------------
+
+external_content_contents = [
+    (NRF_BASE / "doc" / "kconfig", "*.rst"),
+]
+
+# Options for zephyr.kconfig -----------------------------------------------------
+
+kconfig_generate_db = True
+kconfig_ext_paths = [ZEPHYR_BASE, NRF_BASE]
+
+# HACK: needed until NCS_* entries issue is resolved
```

Comment:
I believe we want to avoid words like `HACK` anywhere in our code base.

I have created a jira regarding improving this, so please adjust comment:
```suggestion
# Adding NCS_ specific entries. Can be removed when the NCSDK-14227 improvement task has been completed.
```

Reply:
adjusted